index.ts

```diff
@@ -44,43 +47,88 @@ export const setupPlugin: BigQueryPlugin['setupPlugin'] = async (meta) => {
         throw new Error('Table ID not provided!')
     }
 
+    if (config.deduplicateEvents in ['true', 'True', 't']) {
```

Comment:
A free text input is not particularly friendly to plugin users - I'd use a `choice` type with `choices` "Yes" and "No", then just do `global.deduplicateEvents = config.deduplicateEvents === "Yes"`

Reply:
Ah, nice, didn't realise `choice` type was possible. Thanks!